libs/radio/shims.d.ts

```diff
@@ -5,6 +5,20 @@
     //% color=#E3008C weight=96 icon="\uf012"
 declare namespace radio {
 
+    /**
+     * Disables the radio for use as a multipoint sender/receiver.
+     * Disabling radio will help conserve battery power when it is not in use.
+     */
+    //% help=radio/off shim=radio::off
+    function off(): void;
```

Comment:
I think this change was auto-generated by your local build, but not sure why that would happen. @abchatra is this change safe?